### PR TITLE
added resposive css mobile

### PR DESCRIPTION
**Pull request by adrianaoliveira-ago**

### CODE REVIEW

File: src/App/BlogPage.css [hunk 1]
```diff
@@ -80,3 +82,43 @@
...
+  }
+  .blog-page-size-video {
+    width: 100%;
+    height: 218px;
+  }
+  .blog-page-videos-videos {
+    flex-direction: column;
+    gap: 10px;
+  }
+  .Blog-page-padding2 {
```
Comment (thiagoribeir015) [inline src/App/BlogPage.css]: classnames should start with lower case

File: src/App/Gallery/index.jsx [hunk 2]
```diff
@@ -37,8 +38,8 @@ function Gallery({ list, title }) {
 
   return (
     <div className="gallery-container">
-      <h2 className="gallery-title">{title}</h2>
-
+      <Title name={title}></Title>
+      {/* <h2 className="gallery-title">{title}</h2> */}
```
Comment (thiagoribeir015) [inline src/App/Gallery/index.jsx:42]: you can remove this commented code

File: src/App/Gyg.jsx [hunk 3]
```diff
@@ -35,7 +36,8 @@ function Gyg() {
 
   return (
     <div>
-      <h2 className="Gyg-title">My favorite Tours</h2>
+      <Title name="My Favorite Tours" />
+      {/* <h2 className="Gyg-title">My favorite Tours</h2> */}
```
Comment (thiagoribeir015) [inline src/App/Gyg.jsx]: you can remove this commented code

File: src/App/Header.css [hunk 4]
```diff
@@ -14,6 +14,7 @@
   position: fixed;
   z-index: 9999;
   -webkit-backface-visibility: hidden;
+  z-index: 1;
```
Comment (thiagoribeir015) [inline src/App/Header.css:17]: there are 2 `z-index` props in this selector, which one should stay?

**Comment by thiagoribeir015** [comment]
Left some comments, fix them and push a new commit with the fixes

File: src/App/BlogPage.css (hunk 1 above)
Comment (adrianaoliveira-ago) [reply src/App/BlogPage.css]: Done

File: src/App/Gyg.jsx (hunk 3 above)
Comment (adrianaoliveira-ago) [reply src/App/Gyg.jsx]: Done

**Pull request merged**